ro-base.obo

```diff
@@ -5861,11 +5910,132 @@ is_a: RO:0002449 ! directly negatively regulates activity of
+property_value: http://purl.org/dc/terms/date "2023-02-28T18:53:32Z" xsd:dateTime
+property_value: IAO:0000112 "phenol (CHEBI:15882) and aniline (CHEBI:17296) are matched molecular pairs because they differ by one chemical transformation i.e., the replacement of aryl primary amine with aryl primary alcohol." xsd:string
+is_symmetric: true
+is_a: RO:0018030 ! chemical relationship
+
+[Typedef]
+id: RO:0018036
+name: is tautomer of
+def: "Two chemicals are tautomers if they can be readily interconverted.\n\nThis commonly refers to prototropy in which a hydrogen's position is changed, such as between ketones and enols. This is also often observed in heterocyclic rings, e.g., ones containing nitrogens and/or have aryl functional groups containing heteroatoms." []
+synonym: "is desmotrope of" EXACT []
+xref: is_tautomer_of
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0003-4423-4370
+property_value: http://purl.org/dc/terms/date "2023-03-18T23:49:31Z" xsd:string
+property_value: IAO:0000112 "3-carboxy-3-mercaptopropanoate (CHEBI:38707) is tautomer of 1,2-dicarboxyethanethiolate (CHEBI:38709) because 3-carboxy-3-mercaptopropanoate is deprotonated on the carboxylic acid whereas 1,2-dicarboxyethanethiolate is deprotonated on the secondary thiol." xsd:string {http://www.w3.org/2000/01/rdf-schema#seeAlso="CHEBI:38707", http://www.w3.org/2000/01/rdf-schema#seeAlso="CHEBI:38709"}
+property_value: seeAlso https://github.com/oborel/obo-relations/issues/697
+is_symmetric: true
+is_a: RO:0018030 ! chemical relationship
+
+[Typedef]
+id: RO:0018037
+name: is substitutent group from
+def: "Group A is a substituent group from Chemical B if A represents the functional part of A and includes information about where it is connected. A is not itself a chemical with a fully formed chemical graph, but is rather a partial graph with one or more connection points that can be used to attach to another chemical graph, typically as a functionalization." []
+xref: is_substituent_group_from
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0003-4423-4370
+property_value: http://purl.org/dc/terms/date "2023-03-18T23:49:31Z" xsd:string
+property_value: IAO:0000112 "carboxylatoacetyl group (CHEBI:58957) is substituent group from malonate(1-) (CHEBI:30795)" xsd:string {http://www.w3.org/2000/01/rdf-schema#seeAlso="CHEBI:30795", http://www.w3.org/2000/01/rdf-schema#seeAlso="CHEBI:58957"}
+property_value: seeAlso https://github.com/oborel/obo-relations/issues/697
+is_a: RO:0018030 ! chemical relationship
+
+[Typedef]
+id: RO:0018038
+name: has functional parent
+def: "Chemical A has functional parent Chemical B if there is chemical transformation through which chemical B can be produced from chemical A. \n\nFor example, the relationship between a salt and a freebased compound is a \"has functional parent\" relationship." []
+xref: has_functional_parent
```

Comment:
?

Reply:
Fixed in the commit [c0ecc1](https://github.com/oborel/obo-relations/pull/728/commits/c0ecc1df96a957bd979bc98214b85b855e6818ad)